bin/license_scout

```diff
@@ -39,3 +39,5 @@ collector.run
 report = collector.issue_report
 
 puts report
+
+exit 2 if !report.empty?
```

Comment:
I'd think that the change that drags this into automate's docker-compose.yml would be tested locally (as one of the goals of the DC based stuff was to be able to run everything locally...)